src/dev/performance/run_performance_cli.ts

```diff
@@ -33,19 +108,35 @@ run(
       throw createFlagError('--journey-path must be an existing path');
     }
 
-    let journeys: Journey[] = [];
+    const journeys: Journey[] = [];
 
     if (journeyPath) {
-      journeys = fs.statSync(journeyPath).isDirectory()
-        ? fs.readdirSync(journeyPath).map((fileName) => {
-            return { name: fileName, path: path.resolve(journeyPath, fileName) };
+      const isDirectory = fs.statSync(journeyPath).isDirectory();
+      if (isDirectory) {
+        const files = fs.readdirSync(journeyPath);
+        journeys.push(
+          ...files.map((file) => {
+            return {
+              name: path.parse(file).name,
+              path: path.resolve(journeyPath, file),
+            };
```

Comment:
IIUC this would not support sub-directories in the `journeyPath`. Is this intended?

Reply:
Good point, didn't actually think about it. I think I should fix it.